libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -342,7 +358,7 @@ AP_GPS_UBLOX::read(void)
                 _delay_time = 750;
             }
         } else {
-            _delay_time = 2000;
+            _delay_time = 4000;
```

Comment:
I strongly nominate we implement that message then instead and push GCS's at it for this implementation. Randomly interrupting our RTCM data messages sounds like a really poor idea and will cause semi erratic behavior, especially if you are also experiencing problems with your radio link.